lib/connection.h

```diff
@@ -564,12 +564,19 @@ public Q_SLOTS:
                         const QString& initialDeviceName,
                         const QString& deviceId = {});
 
+    //! \brief Use an existing access token to connect to the homeserver
+    //!
+    //! This is the old signature that will be deprecated in 0.8; since 0.7.2,
+    //! device id is resolved from the access token. Use 2-arg assumeIdentity()
+    void assumeIdentity(const QString& mxId, const QString& accessToken,
```

Comment:
`[[deprecated]]`?

Reply:
That would make impossible for the code that tries to stay compatible with pre-0.7.2 API to compile without deprecation warnings; but I agree, should be a minor nuisance.